pkg/filetree/depth_first_path_walker.go

```diff
@@ -59,11 +59,12 @@ func (w *DepthFirstPathWalker) Walk(from file.Path) (file.Path, *filenode.FileNo
 	w.pathStack.Push(from)
 
 	var currentPath file.Path
-	var currentNode *filenode.FileNode
+	var currentNode *nodeAccess
```

Comment:
nit: style guide shows `var ()` for multiple declarations, but we've never really mentioned it before now:
https://google.github.io/styleguide/go/best-practices#variable-declarations